json-rpc/spec.json

```
@@ -196,6 +196,94 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
+			"params": [
+				{
+					"name": "blockCount",
+					"description": "Number of blocks in the requested range (positive integer; less may be returned if requesting the entire specified range is not supported).",
```

```suggestion
					"description": "Number of blocks in the requested range (less may be returned if requesting the entire specified range is not supported).",
```
The schema should denote the type.

It's also not clear to me what "not supported" means?